src/librbd/deep_copy/ObjectCopyRequest.cc

```diff
@@ -342,7 +342,9 @@ void ObjectCopyRequest<I>::send_write_object() {
   librados::ObjectWriteOperation op;
   uint64_t buffer_offset;
 
-  if (!m_dst_image_ctx->migration_info.empty()) {
+  bool migration = ((m_flags & OBJECT_COPY_REQUEST_FLAG_MIGRATION) != 0);
+  if (migration) {
+    ldout(m_cct, 20) << "assert_snapc_seq=" << dst_snap_seq << dendl;
```

Comment:
Nit: this line is not in the original commit.

Reply:
It was suggested during resolution and I thought it would not hurt to backport it too here. It looks useful (we already have debug messages when adding other ops, and only this one was missing). Or do you see any problems with this? I then will remove.

Reply:
No issues, it just caught my eye because the lines didn't match up.